.github/workflows/scripts/sensitive_file_check.py

```
@@ -0,0 +1,149 @@
+        config_path (str): Path to the configuration file.
+
+    Returns:
+        list: A list of compiled regex objects.
+
+    Raises:
+        FileNotFoundError: If the config file does not exist.
+        re.error: If a regex pattern is invalid.
+    """
+    patterns = []
+    try:
+        with open(config_path, "r", encoding="utf-8") as file:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)